### DISCUSSION TITLE
Import Numpy Array as mesh with RGB values?

### DISCUSSION THREAD

**Discussion by PMDevine1** [cnr-isti-vclab/PyMeshLab#154] [Ideas]
Hello,

I am trying import a point cloud into PyMeshLab via numpy array.  Setting the xyz vertices is straight forward, but I don't see the possibility to import vertex RGB values from the numpy array.  Am I missing something or is this something that could possibly be added in a future release? 

Thanks for everything you do -- PyMeshLab has been great to use.

-Phil

**Comment by alemuntoni** [comment]
This feature is not available in 2021.7 version, but has been already implemented and will be available in the next version of pymeshlab.

**Reply by PMDevine1** [reply]
That's great, thanks!
